pages/coinbase/signin.py

```diff
@@ -49,20 +49,19 @@ def __init__(self, driver, spawn=False, bypass_auth=True):
 
     # BODY
 
-    email_field_loc =               (by.ID, 'email')
-    pass_field_loc =                (by.ID, 'password')
-    stay_signed_in_checkbox_loc =   (by.ID, 'stay_signed_in')
-    sign_in_button_loc =            (by.ID, 'signin_button')
-    forgot_password_link_loc =      (by.LINK_TEXT, 'Forgot password?')
-    no_account_link_loc =           (by.LINK_TEXT, "Don't have an account?")
-    privacy_policy_link_loc =       (by.PARTIAL_LINK_TEXT, 'Privacy Policy')
+    email_field_loc =                  (by.ID, 'email')
+    pass_field_loc =                   (by.ID, 'password')
+    keep_me_signed_in_checkbox_loc =   (by.ID, 'stay_signed_in')
+    sign_in_button_loc =               (by.ID, 'signin_button')
+    forgot_password_link_loc =         (by.LINK_TEXT, 'Forgot password?')
+    not_have_account_link_loc =        (by.LINK_TEXT, "Don't have an account?")
```

Comment:
no_account_link was better. If you can use exact text (like above, "keep_me_signed_in" isn't FULL text, but it still makes sense so okay), do that, but if exact text doesn't make sense using a description is best.

Reply:
I'm confused which one is better to use?

Reply:
no_account_link_loc was better.